themes/edx.org/lms/static/js/demographics-collection.js

```diff
@@ -1,23 +0,0 @@
-$(document).ready(function() {
```

Comment:
@Tj-Tracy I had removed this file when I ported the API call and dismiss functionality to the DemographicsBanner React component.

Does this mean I will have broken the functionality for the "classic" banner? It seems like I might not want to do this cleanup yet and add this file back until we are ready to cutover.

Reply:
Yeah that will break the dismiss. Would you mind adding it back?

Reply:
On it.